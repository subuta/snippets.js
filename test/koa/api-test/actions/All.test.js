/* global expect, describe, it */

import { build, format, snippets as s } from 'bld.js'

import AllTest from 'lib/koa/api-test/actions/All'

describe('koa/api-test/actions/All', () => {
  it('should create koa Index(All) action test', () => {
    expect(format(AllTest({
      model: 'user',
      config: {}
    }))).toMatchSnapshot()
  })
})
