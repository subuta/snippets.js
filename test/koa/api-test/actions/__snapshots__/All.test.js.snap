// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`koa/api-test/actions/All should create koa Index(All) action test 1`] = `
"test('index should list user', async (t) => {
  const {request} = t.context

  // mock jwks
  const token = createToken(privateKey, '123', currentUser)
  jwksEndpoint('http://localhost', [{pub: publicKey, kid: '123'}])

  const response = await request
    .get('/api/users')
    .set('Authorization', \`Bearer \${token}\`)

  t.is(response.status, 200)
  t.deepEqual(response.body.length, 3)
  t.deepEqual(response.body, [2054, 62371, 41854])
})
"
`;
